CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Add `ruby-3.0` to CI
 * Add `codecov` branch coverage check in CI
+* Add methods which colorize log on red and green
```

Comment:
Add exact method names in readme like `OnlyofficeLoggerHelper.green_log` in CHANGELOG